griffin/secret_management_util.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+import hvac
+
+from griffin.griffin.decorators import singleton
+from .config import ENTITY_NAME, ENV, REGION, SERVICE_NAME, HASHICORP_URL, AWS_INSTANCE, VAULT_TOKEN, MOUNT_POINT, KEY
+from .exceptions import VaultAuthenticationException, SecretNotFoundException, ValidationFailedException
+
+LOGGER = logging.getLogger("secret_management_util")
+
+
+@singleton
+class SecretManagementUtil:
+    """
+    This util is responsible for making connection with HashiCorp Vault service and to fetch the secrets
+    from the Vault
+    """
+
+    @staticmethod
+    def _validate_values_from_input(entity_name, service_name):
```

Comment:
Why do we need to pass entity name and service to this function. Shouldn't we fetch is directly from the environment?

Reply:
yes we can fetch directly from env
changes are done, please check